cmake/modules/RootMacros.cmake

```diff
@@ -2114,6 +2114,48 @@ function(ROOT_APPEND_LIBDIR_TO_INSTALL_RPATH target install_dir)
   set_property(TARGET ${target} APPEND PROPERTY INSTALL_RPATH "${new_rpath}")
 endfunction()
 
+#----------------------------------------------------------------------------
+# If path is a system path, set the return variable is_system_path to true.
+# The 1st argument is the path that should be checked
+# The 2nd argument is the return value
+#----------------------------------------------------------------------------
+function (IS_SYSTEM_PATH path is_system_path)
+  foreach (dir ${CMAKE_SYSTEM_PATH})
```

Comment:
I don't think this variable is part of (the official) CMake, so this might always be false.